roles/openjdk-8/tasks/main.yml

```diff
@@ -25,3 +35,19 @@
       name: openjdk-8-jdk
       update_cache: yes
     when: ansible_distribution == 'Debian'
+
+  # The repo's release file is expired which will cause errors when updating apt cache
+  - name: remove backports repo (Debian Jessie)
+    apt_repository:
+      repo: "{{ jessie_backports_repo }}"
+      state: absent
+    changed_when: false
+    when: ansible_distribution == 'Debian' and ansible_distribution_release == 'jessie'
+
+  - name: disallow expired release files (Debian Jessie)
+    lineinfile:
+      path: /etc/apt/apt.conf
+      regexp: '^Acquire::Check-Valid-Until'
```

Comment:
This line `Acquire::Check-Valid-Until` has used multiple times, do you think we should store it in a variable?